### ISSUE TITLE
torch.no_grad() for evaluation mode

### ISSUE THREAD

**Issue by marjanfamili** [alan-turing-institute/affinity-vae#113]
with torch.no_grad(): for evaluation mode to speed up the program
